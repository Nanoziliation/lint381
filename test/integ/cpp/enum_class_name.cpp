enum class bar {};

enum class Bar_e {};
