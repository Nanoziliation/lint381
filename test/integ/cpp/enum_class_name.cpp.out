enum_class_name.cpp:1:12: error: Enum class name 'bar' should be capitalized
enum class bar {};
           ^^^
enum_class_name.cpp:3:12: error: Enum class name 'Bar_e' shouldn't end with '_e'
enum class Bar_e {};
           ^^^^^
